aws/iac/ec2.rego

```diff
@@ -92,7 +92,9 @@ default ec2_public_ip = null
 aws_issue["ec2_public_ip"] {
     resource := input.Resources[i]
     lower(resource.Type) == "aws::ec2::instance"
-    resource.Properties.NetworkInterfaces[_].AssociatePublicIpAddress == true
+    nic := resource.Properties.NetworkInterfaces[_]
+    nic.AssociatePublicIpAddress == true
+    lower(resource.SecurityGroups[_]) == "default"
```

Comment:
will be like resource.Properties.SecurityGroups[] not resource.SecurityGroups[]

Reply:
Yes, fixed

Reply:
this line should be like `lower(resource.Properties.SecurityGroups[_]) == "default"`